src/LoadableModules.cc

```diff
@@ -8,29 +8,30 @@
 
 #include "squid.h"
 #include "debug/Messages.h"
-#include "debug/Stream.h"
 #include "LoadableModule.h"
 #include "LoadableModules.h"
-#include "wordlist.h"
+#include "sbuf/List.h"
 
 static void
-LoadModule(const char *fname)
+LoadModule(const SBuf &fname)
 {
     debugs(1, DBG_IMPORTANT, "Loading Squid module from '" << fname << "'");
 
-    LoadableModule *m = new LoadableModule(fname);
+    auto *m = new LoadableModule(fname);
     m->load();
     debugs(1, 2, "Loaded Squid module from '" << fname << "'");
 
     //TODO: TheModules.push_back(m);
 }
 
 void
-LoadableModulesConfigure(const wordlist *names)
+LoadableModulesConfigure(const SBufList &names)
 {
-    int count = 0;
-    for (const wordlist *i = names; i; i = i->next, ++count)
-        LoadModule(i->key);
+    unsigned int count = 0;
```

Comment:
This PR does not have to change this type, but since it does, let's use a more appropriate type for counting things inside a container:

```suggestion
    size_t count = 0;
```

Alternatively, remove `count` and just report names.size(), simplifying this code further!

Reply:
done in [fd858aa](https://github.com/squid-cache/squid/pull/1738/commits/fd858aaa13035510319477ae023f7064982d4576)